dashboards/haproxy/haproxy-prometheus.json

```diff
@@ -32,25 +32,31 @@
             },
             "dataSets": [
               {
+                "breakdowns": [],
+                "dimensions": [],
+                "measures": [],
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "prometheusQuery": "haproxy_frontend_http_responses_total{${Cluster},${Location},${Namespace}}"
+                  "prometheusQuery": "haproxy_frontend_http_responses_total{${Cluster},${Location},${Namespace}}",
```

Comment:
I think this should be wrapped in `rate(` so it's showing the current responses per second.

As-is, I think this is showing a running total of responses, which is why a lot of graphs are trending up in the screenshot, even though the actual traffic is probably not increasing. This probably applies to most the queries in here on `..._total` metrics, which are likely to be cumulative counter metrics.